tools/schema_tools/gen_output.py

```diff
@@ -3,6 +3,7 @@
 import MySQLdb as mysqldb
 import os
 from socket import gethostname
+from lsst.sims.utils import photo_m5
```

Comment:
I thought that we were supposed to call this function m5_flat_sed because of the particular assumptions made. This name gives the impression of a full photometric m5.